pkg/autoupgrade/daemon_test.go

```diff
@@ -471,6 +474,10 @@ func TestDaemonSync(t *testing.T) {
 			app.Spec.AutoUpgrade = ptrTrue
 			app.Spec.Image = "acorn/acorn-enabled:latest"
 		case "notify-app":
+			app.Spec.AutoUpgrade = ptrTrue
+			app.Spec.NotifyUpgrade = ptrTrue
+			app.Spec.Image = "acorn/acorn-notify:latest"
+		case "notify-app-alone":
```

Comment:
I think the test cases in the table test below this switch need to be updated to add this new `"notify-app-alone"` case.

Reply:
So all of these test cases are actually testing all of the apps defined on line 466. As such, I explicitly did not add a case to test the `notify-app-alone` app since it should not appear in any of the results